dobi.yaml

```diff
@@ -615,6 +615,27 @@ job=run-analyzer-deployer-integration-tests:
   depends:
     - integration-env
 
+job=run-analyzer-executor-integration-tests:
+  use: analyzer-executor-build
+  net-mode: grapl-network
+  command: |
+    /bin/bash -c "
+      source venv/bin/activate &&
+      cd analyzer_executor && 
+      export PYTHONPATH="${PYTHONPATH}:$(pwd)/src" &&
```

Comment:
Looks like the double-quote here escapes and breaks the `command`? Prob want single quotes.

Reply:
switching the inner quotes to `"`s actually broke this, switching outer quotes to single and inner quotes to double works

Reply:
i am very confused as to how/why this works

Reply:
oh nvm you fixed it

Reply:
I was also very confused why it worked before!  YAML weirdness, or PYTHONPATH is quoted?  I didn't dive into it.